### PR TITLE
CITATION.cff file

### PR DESCRIPTION
**Pull request by dr-shorthair**
Related to #205

### CODE REVIEW

**Comment by situx** [comment]
I always understood cff to Give one version to cite for a whole repository. Should the content of the cff be the latest version of GeoSPARQL or are there options to also cite different versions?

**Comment by dr-shorthair** [comment]
Yes, .cff is for the repo as a whole :-( 
Since the doc is to be issued as a standalone, discrete product, should it have its own repo? 
i.e. GitHub hygiene is 
- one product == one repo => one citation ...

**Comment by situx** [comment]
It is probably a handy shortcut for people wanting to cite the latest draft, so if the cff resembles the latest version that would be fine with me. For the other cases I have proposed bibtex files. I am not sure if we will create another repo just for 1.1.

File: CITATION.cff [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  given-names: "Joseph"
+  orcid: "https://orcid.org/0000-0001-5075-6234"
+- family-names: "Cox"
+  given-names: "Simon J D"
+  orcid: "https://orcid.org/0000-0002-3884-3420"
+- family-names: "Bonduel"
+  given-names: "Matthias"
+  orcid: "https://orcid.org/0000-0002-3313-924X"
+- family-names: "Knibbe"
+  given-names: "Frans"
```
Comment (FransKnibbe) [inline CITATION.cff:24]: I've got an ORCID too: https://orcid.org/0000-0003-3789-2260

**Comment by jabhay** [comment]
Another ORCID: http://orcid.org/0000-0001-5075-6234

**Comment by mperry455** [comment]
My ORCID: https://orcid.org/0000-0003-1988-462X

**Review by nicholascar — APPROVED** [review]

**Comment by nicholascar** [comment]
Missing ORCIDs added (John Herring doesn't have one) so now approved and I'll  mrege without a second approval as we've discussed this in the meeting just ended.

**Pull request merged**